### DISCUSSION TITLE
When to use Transients API and how

### DISCUSSION THREAD

**Discussion by bph** [WordPress/developer-blog-content#29] [Topic Ideas]
Thank you @Milana for suggesting this topic. 
When you have a minute, maybe you can elaborate a bit on what this blog could cover.

**Comment by ryanwelcher** [comment]
I think I'd like to take this one on.

I can see this as an article that explains how the Transients API works, how it interacts with object caching in WordPress, and differences between using `get/set_transien`t and `wp_cache_get/set`.

**Comment by bph** [comment]
@ryanwelcher Thank you for stepping up to the plate, so to speak!

**Comment by bph** [comment]
@ryanwelcher Seems this hasn't been approved, yet. We put it in the hopper for the Feb 1st meeting, but it shouldn't keep you from working on an it.

**Comment by bph** [comment]
The topic was approved and discussion here is closed.
Please continue the conversation on the issue

- #213

**Comment by ryanwelcher** [comment]
The article has been published - https://developer.wordpress.org/news/2024/06/11/an-introduction-to-the-transients-api/
